### DISCUSSION TITLE
Include namespaces in template output if "createNamespace: true"

### DISCUSSION THREAD

**Discussion by yellowmegaman** [helmfile/helmfile#248] [Ideas]
Hi!
We're using helmfile to prepare all the yaml's to be consumed by GitOps tool.

Does it make sense to create PR that adds namespaces yaml definition to `helmfile template` if `createNamespace: true` is set?

We definitely need this

Thanks in advance for the info!

**Comment by jouve** [comment]
hi,

createNamespace is passed to helm (https://github.com/helmfile/helmfile/blob/main/pkg/state/state.go#L2469), so I think this should be discussed there :)

**Comment by yellowmegaman** [comment]
Oh, right. Thanks for pointing that out!
